samples/Mvc.ActionConstraintSample.Web/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc;
+
+namespace ActionConstraintSample.Web.Controllers
```

Comment:
Odd for namespace and directory to be different. Suggest making both `Mvc.ActionConstraintSample` i.e. drop the `Web` suffix since the `Mvc` prefix implies the project type. Same for the other moved projects.